features/employers/benefit_applications/admin_terminating_benefit_application_steps.rb

```diff
@@ -57,9 +57,10 @@
   find('.plan-year-submit', text: 'SUBMIT', wait: 5).click
 end
 
-And(/^employer clicks OK in warning modal$/) do
+And(/^employer clicks (.*) in warning modal$/) do |button_text|
   # find('.swal-button swal-button--confirm swal-button--danger', text: "OK").click
-  click_button 'OK'
+  find_button(button_text).visible?
```

Comment:
the goal here is to assert on a specific button being present and not on any text that the button contains

while this is now more generic than the previous step definition, it still uses the text to assert on which we don't want to be doing